### ISSUE TITLE
Community survey

### ISSUE THREAD

**Issue by jooola** [LibreTime/libretime#1263]
#### TLDR;
Please fill the community survey at https://discourse.libretime.org/t/libretime-community-survey/972
Or directly at https://framaforms.org/libretime-community-survey-1626857592

Thanks !

---
Original issue:

So before posting this ticket on Discourse I wanted to gather some input from the maintainers/community so we agree on what we want to ask with this survey. And please tell me if I can improve the wording, or if the text has some ugly typos/mistakes.

This is a child ticket of #1247 

Do we want to use a survey platform to gather the data or should we simply compile the answers on Discours ?

cc @Robbt @paddatrapper 

---

Greetings Libretime community,

In order to get a better idea of how and by whom Libretime is used, we would like to get feedback from the community. This will help us to focus on features that really matter. This idea of a survey mostly comes from not knowing whether to drop support for the SysV init system or the Upstart init system.

## Environments

- Which distribution is Libretime installed on ? **Distribution name** and **version** (e.g. `Ubuntu 18.04` or `Centos 8.0`)
  - If it wasn't your favorite distribution, which one would you like to be supported ?

- Which version of Libretime or Airtime do you use ?  (e.g. `Airtime 2.5`, `Libretime 3.0.0-alpha.9` or commit hash `d58fe19ae3cd01f6339e8f25d76b607e5d6b3716`)

- Which init system do you use on this system ? (e.g. `SystemD`, `SysV`, `Upstart`)
  - If it isn't SystemD, would you mind moving to SystemD ?

## Usage

- Who is managing Libretime ? (e.g. a dedicated system administrator or a person without much IT knowledge)
  - Who installed the Libretime instance ?
  - Who is managing the Libretime settings or users ?

- Is your Libretime installed  as standalone or part of a larger ecosystem ? (e.g. `Wordpress`, `LDAP` for shared identity across services, ...)

- Which installation method was used to install the Libretime instance ? (eg. the `./install` script, `apt`, `docker`, custom homemade script, orchestration tool)

## Long term 

- Would you like to see Libretime have a embedded telemetry system, in order to gather the **environements** information above ?
- Would you like to see Libretime have a small survey during the installation process to gather the **usage** information above ?

**Comment by paddatrapper** [comment]
I think using a survey platform of some kind would be helpful. I can create a Quick Survey instance (MIT Licence) on storm.debian.net that can then be populated with questions?

On the questions:

* What version are you using - perhaps add an example of a git commit hash, as some people will not be tracking release numbers
* Long term questions: Either `Could you see...` or `Would you like to see...`

**Comment by jooola** [comment]
Here is the survey I created https://framaforms.org/libretime-community-survey-1626857592

I'll publish the survey on discourse pretty soon, so people are able to fill it.

Tell if something is wrong, so I can fix it.

**Comment by jooola** [comment]
I published a topic on discourse https://discourse.libretime.org/t/libretime-community-survey/972

Could an admin pin the topic on discourse so everyone sees the survey ?

**Comment by paddatrapper** [comment]
The only issue I see is that the next button and some of the default options are in French. Not a show-stopper though, as it is fairly clear what they mean even when you can't speak French

**Comment by jooola** [comment]
Yes, my bad, I haven't found any English trustworthy survey service. I hope people don't mind, otherwise I am happy to move the survey on the provider of your choice.
Or you told me about setting up a survey instance on storm.debian.net (I don't know what it is).

**Comment by paddatrapper** [comment]
I'm happy to leave it as is

**Comment by jooola** [comment]
@paddatrapper Could you pin the Survey on discourse, so it doesn't get pushed down by more recent topics, and it will "promote" as "official survey" and not some random Libretime user survey.

**Comment by paddatrapper** [comment]
I don't have admin on the discourse site, so I can't. @Robbt or @hairmare perhaps can?

**Comment by jooola** [comment]
Some references for a future "telemetry" ticket:

https://github.com/audacity/audacity/pull/835
https://cleaninsights.org/

**Comment by jooola** [comment]
The community survey will be closed on the next community meeting. The survey hasn't receive much participation for the past few weeks anyway.

**Comment by jooola** [comment]
I am finally closing the survey, new survey can be made with additional question some other time.

Here is the result of the survey.

![image](https://user-images.githubusercontent.com/19195485/148351995-2549ebb7-38fa-4e94-b5ea-0013ebf27da7.png)
![image](https://user-images.githubusercontent.com/19195485/148352051-2ae179e0-d64e-4c6e-b804-7339b0e38db0.png)
![image](https://user-images.githubusercontent.com/19195485/148352073-45b23ec3-fb84-42cc-8817-40ebda0e8287.png)
![image](https://user-images.githubusercontent.com/19195485/148352152-b1c22445-7050-4238-b503-95b14978748f.png)
![image](https://user-images.githubusercontent.com/19195485/148352186-a61374e4-1397-42ac-9c2d-e8b1523b3382.png)
![image](https://user-images.githubusercontent.com/19195485/148352208-65dc3d87-e131-440f-afee-1ead5a082f05.png)

Data: 
[libretime_community_survey.csv](https://github.com/LibreTime/libretime/files/7820431/libretime_community_survey.csv)
